CHANGELOG.md

```diff
@@ -4,6 +4,11 @@
 
 - Fix image URLs for suggest endpoints [#2761](https://github.com/opendatateam/udata/pull/2761)
 - Switch from `Flask-restplus` to its fork `Flask-rest-x` [2770](https://github.com/opendatateam/udata/pull/2770)
+- :warning: Upgrade to `Flask-Security-Too` version 4.0.0 [#2772](https://github.com/opendatateam/udata/pull/2772):
```

Comment:
Somehow, Flask-Security now validates email checking that domain name exist even on login.
This leads to test failure, ex: https://app.circleci.com/pipelines/github/opendatateam/udata/2382/workflows/ee5bcf3f-9af9-4165-8242-a4271cc18cbe/jobs/23841/tests#failed-test-0
(You can try to create a user with email=test@this-is-a-non-existant-domain.org with the cli and then log in to reproduce)

We may want to add this check, but should make sure it won't prevent existing users to login.